src/routes/userRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router = express.Router();
+const userController= require('../controllers/userController');
+const { verifyToken } = require('../middlewares/auth');
+
+router.post('/adminsignup',userController.adminsignup)
+router.post('/kakaologin',userController.kakaologin)
+router.post('/adminlogin',userController.adminlogin)
+
+module.exports = {
+    router
```

Comment:
```suggestion
  router
```
JavaScript의 기본 컨벤션은 띄어쓰기 두 칸입니다